### ISSUE TITLE
Missing pandas_flavor install

### ISSUE THREAD

**Issue by jdakka** [bp-kelley/descriptastorus#6]
```
>>> from descriptastorus.descriptors import rdNormalizedDescriptors

ERROR:root:Unable to make new descriptors, descriptor generator not installed
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/home/dakka/miniconda3/envs/ligand_ml/lib/python3.6/site-packages/descriptastorus/descriptors/__init__.py", line 2, in <module>
    from .DescriptorGenerator import *
  File "/home/dakka/miniconda3/envs/ligand_ml/lib/python3.6/site-packages/descriptastorus/descriptors/DescriptorGenerator.py", line 35, in <module>
    import pandas_flavor as pf
ModuleNotFoundError: No module named 'pandas_flavor'
```

**Comment by bp-kelley** [comment]
This was fixed in #5 

a pip install using this requirements file should fix the issue.  It looks like we may have overlooked setup.py though.  Thanks for the issue.

**Comment by jdakka** [comment]
Reopening this as the dependency for pandas-flavor is not included in setup.py

**Comment by bp-kelley** [comment]
Thanks, this was fixed in commit 0f0c2a9f60885e67f8f28f557526b0b4e2be03b5
